exporters/prometheus/exporter_test.go

```diff
@@ -521,6 +523,41 @@ func TestPrometheusExporter(t *testing.T) {
 		{
 			name:         "counter utf-8",
 			expectedFile: "testdata/counter_utf8.txt",
+			options: []Option{
```

Comment:
It's probably a good idea to add namespace to most (if not all) test cases, just to make sure it's also covered

Reply:
I added namespace to a few... I'm not sure we need more to cover all the cases? let me know what  you think.